package.json

```diff
@@ -100,7 +100,8 @@
     "swagger-jsdoc": "^6.2.8",
     "swagger-ui-express": "^5.0.0",
     "winston": "^3.3.3",
-    "winston-mail": "^2.0.0"
+    "winston-mail": "^2.0.0",
+    "axios": "^1.7.2"
```

Comment:
Adding a dependency is not a lightweight change as it introduces security risks and maintenance burden. It seems this dependency is used only for 3 HTTP calls. Please use the native `http` module or one of the existing dependencies.
Please also use `npm install --save` to maintain alphabetical order and avoid future churn.

Reply:
This is solved and will be available when the pull request will be updated. Axios was removed and was replaced with node-fetch (already used in the project).